src/Components/ExternalResult/ResultList.tsx

```diff
@@ -220,7 +218,7 @@ export default function ResultList() {
           {value}
           <i
             className="fas fa-times ml-2 rounded-full cursor-pointer hover:bg-gray-500 px-1 py-0.5"
-            onClick={() => removeFilter(paramKey)}
+            onClick={(_) => removeFilter(paramKey)}
```

Comment:
```suggestion
            onClick={() => removeFilter(paramKey)}
```
If event (_) is an optional one, so its better to be avoided when we don't use them